packages/components/src/CoronaBanner/banner.js

```diff
@@ -0,0 +1,139 @@
+import React, { Fragment } from "react"
+import CoronaSvg from "../../../../images/covid-virus-icon.svg"
+import Chevron from "../../../../images/chevron.svg"
+import "../../../../less/corona-banner.less"
+
+const LinkWrapper = ({ condition, wrapper, children }) =>
+  condition ? wrapper(children) : children
+
+const CoronaBanner = ({
+  newCases,
+  hospitalised,
+  deaths,
+  vaccinated,
+  vaccinatedPercentage,
+  siteUrl,
+  showLinks,
+}) => {
+  const showVaccinatedPercentage = () => {
+    return vaccinated ? (
+      <span className="stat-percent">({vaccinatedPercentage}%)</span>
+    ) : (
+      <span className="stat-percent">{vaccinatedPercentage}%</span>
+    )
+  }
+
+  return (
+    <div className="corona-banner-wrapper">
+      <LinkWrapper
+        condition={showLinks}
+        wrapper={children => (
+          <a href={siteUrl} target="_parent">
+            {children}
+          </a>
+        )}
+      >
+        <Fragment>
+          <div className="corona-container">
+            <div className="content-container">
+              <header className="container-header">
+                <h1 className="banner-title">
+                  Covid-19 <br /> i Finland
+                </h1>
+                <img className="virus-image" src={CoronaSvg} alt="" />
+              </header>
+              {newCases !== null && (
+                <div className="stat">
+                  <div className="stat-value">{newCases}</div>
+                  <div className="stat-label">nya fall</div>
+                </div>
+              )}
+              {hospitalised !== null && (
+                <div className="stat mobile-hidden">
+                  <div className="stat-value">{hospitalised}</div>
+                  <div className="stat-label">på sjukhus</div>
+                </div>
+              )}
+              {deaths !== null && (
+                <div className="stat smallest-screens-hidden">
+                  <div className="stat-value">{deaths}</div>
+                  <div className="stat-label">dödsfall</div>
+                </div>
+              )}
+              {(vaccinated !== null || vaccinatedPercentage) && (
```

Comment:
Do we need this condition in the end?

Reply:
Because in line 66 we are rendering it regardless of what value it has, and in `showVaccinatedPercentage` we actually do some comparison on it too.

Reply:
I was trying to make it so that if the vaccinated amount is missing but we have the percentage, then only percentage shows without parentheses around the value. But now I see what you mean, I will clean it up and make a new try!

Reply:
My goal is to cover all 4 possible outcomes:

1. amount === null && percentage === null
(<div className="stat"> element won't render at all)

2. amount !== null && percentage !== null
(<div className="stat"> element will render and inside it <div className="stat-value">  will show amount and inside parentheses percentage number and sign, and <div className="stat-label"> will render)

3. amount !== null && percentage === null
(<div className="stat"> element will render and inside it <div className="stat-value"> will only show amount but not percentage number or sign or parentheses and <div className="stat-label"> will render)

4. amount === null && percentage !== null
(<div className="stat"> element will render and inside it <div className="stat-value"> will only show percentage number and sign without parentheses and <div className="stat-label"> will render)

Reply:
Right, I see. Can we check for truthy values in line 63 or do we need to check explicitly for `not null` (`vaccinated || vaccinatedPercentage`)?

Also, I wish we had pattern matching in javascript 😅